package.json

```diff
@@ -53,19 +55,36 @@
     ]
   },
   "devDependencies": {
+    "@babel/core": "^7.6.4",
+    "@babel/preset-env": "^7.6.3",
+    "@babel/preset-react": "^7.0.0",
     "@storybook/addon-actions": "^5.2.5",
     "@storybook/addon-links": "^5.2.5",
     "@storybook/addon-viewport": "^5.2.5",
     "@storybook/addons": "^5.2.5",
     "@storybook/react": "^5.2.5",
     "@storybook/storybook-deployer": "^2.8.1",
-    "babel-loader": "^8.0.6",
+    "@typescript-eslint/eslint-plugin": "2.x",
+    "@typescript-eslint/parser": "2.x",
+    "babel-eslint": "10.x",
     "codecov": "^3.6.1",
     "enzyme": "^3.10.0",
     "enzyme-adapter-react-16": "^1.14.0",
-    "eslint-config-prettier": "^6.3.0",
+    "eslint": "6.x",
+    "eslint-config-prettier": "^6.5.0",
+    "eslint-config-react-app": "^5.0.2",
+    "eslint-plugin-flowtype": "3.x",
+    "eslint-plugin-import": "2.x",
+    "eslint-plugin-jsx-a11y": "6.x",
```

Comment:
Ideally, we shouldn't list any of these in our `package.json` file, they should be tucked away in `react-scripts`, and I still don't fully agree on adding them here, because we'll end up maintaining them manually which beats the points of using `react-scripts`, I'd rather investigate more why we're getting those warnings in the first place.
Since, as far as I can recall, we didn't get them since the beginning.

Reply:
```
arn install v1.19.1
[1/4] Resolving packages...
[2/4] Fetching packages...
info fsevents@2.0.7: The platform "linux" is incompatible with this module.
info "fsevents@2.0.7" is an optional dependency and failed compatibility check. Excluding it from installation.
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
[3/4] Linking dependencies...
warning " > @apollo/react-hooks@3.1.2" has unmet peer dependency "apollo-client@^2.6.4".
warning "@apollo/react-hooks > @apollo/react-common@3.1.2" has unmet peer dependency "apollo-client@^2.6.4".
warning "@apollo/react-hooks > @apollo/react-common@3.1.2" has unmet peer dependency "apollo-utilities@^1.3.2".
warning " > babel-loader@8.0.6" has unmet peer dependency "@babel/core@^7.0.0".
warning " > babel-loader@8.0.6" has unmet peer dependency "webpack@>=2".
warning "@storybook/react > @babel/preset-react@7.0.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > react-table@7.0.0-beta.12" has unmet peer dependency "prop-types@^15.5.0".
warning " > @storybook/react@5.2.5" has unmet peer dependency "@babel/core@^7.0.1".
warning "@storybook/react > @babel/plugin-transform-react-constant-elements@7.6.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@storybook/react > @babel/preset-flow@7.0.0" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@storybook/react > @babel/preset-flow > @babel/plugin-transform-flow-strip-types@7.6.3" has unmet peer dependency "@babel/core@^7.0.0-0".
warning "@storybook/react > @storybook/core > @babel/plugin-proposal-object-rest-spread@7.6.2" has unmet peer dependency "@babel/core@^7.0.0-0".
warning " > eslint-config-prettier@6.3.0" has unmet peer dependency "eslint@>=3.14.1".
warning " > eslint-plugin-prettier@3.1.1" has unmet peer dependency "eslint@>= 5.0.0".
```
Seems to be related to storybook and @apollo/react-hooks? At least storybook uses it's own version of webpack and keeping that aligned with react-scripts should be maintained within this project or something similar?